.editorconfig

```diff
@@ -0,0 +1,21 @@
+; indicate this is the root of the project
+root = true
+
+[*]
+charset = utf-8
+
+end_of_line = LF
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+indent_style = space
+indent_size = 4
+
+[Makefile]
+indent_style = tab
+
+[makefile]
+indent_style = tab
+
+[*.go]
+indent_style = tab
```

Comment:
hard tabs? :(

Reply:
Hard tabs is the standard for go and using `gopls` and `goimports` will autoformat it that way.

Seeing I tend to sway towards the language community on things, that is where I will go with `*.go` files for consistency and to not push away contributors